src/test/org/codehaus/groovy/tools/stubgenerator/GROOVY9801.groovy

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Licensed to the Apache Software Foundation (ASF) under one
+ *  or more contributor license agreements.  See the NOTICE file
+ *  distributed with this work for additional information
+ *  regarding copyright ownership.  The ASF licenses this file
+ *  to you under the Apache License, Version 2.0 (the
+ *  "License"); you may not use this file except in compliance
+ *  with the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+package org.codehaus.groovy.tools.stubgenerator
+
+class GROOVY9801 extends StringSourcesStubTestCase {
+
+    Map<String, String> provideSources() {
+        [
+                'Interface9801.groovy': '''
+                    interface Interface9801 {
+                        default String method() { return 'test' }
+                    }
+                ''',
+                'Class9801.groovy': '''
```

Comment:
The extension of the file should be 'java'?

Reply:
Yea, that's probably a better test. Maybe I don't even need to be testing this, it's not part of the stub generation.